mealie/run.sh

```diff
@@ -5,20 +5,6 @@ set -e
 # Get Reload Arg `run.sh reload` for dev server
 ARG1=${1:-production}
 
-# Get PUID/PGID
-PUID=${PUID:-911}
-PGID=${PGID:-911}
-
-add_user() {
-    groupmod -o -g "$PGID" abc
-    usermod -o -u "$PUID" abc
-
-    echo "
-    User uid:    $(id -u abc)
-    User gid:    $(id -g abc)
-    "
-    chown -R abc:abc /app
-}
```

Comment:
Why isn't this needed anymore? Don't we still need to set permissions on the mounted directories?